db/workflow.go

```diff
@@ -109,9 +109,10 @@ func insertActionList(ctx context.Context, db *sql.DB, yamlData string, id uuid.
 
 		workerID, err := getWorkerID(ctx, db, task.WorkerAddr)
 		if err != nil {
+			if err == sql.ErrNoRows {
```

Comment:
this should be `if errors.Cause(err) == sql.ErrNoRows {` right?

Reply:
Right... I put the check for errors.Cause somewhere else 🤦‍♀️